horace_core/sqw/coord_transform/@aProjection/private/check_3vector_.m

```diff
@@ -0,0 +1,21 @@
+function val = check_3vector_(val)
+% Verify input 3-vector describing the direction can be considered valid
```

Comment:
the return value is transposed but the name is just check. the user should be aware from the op-name that the input is changed

Reply:
I've renamed it and not sure the modification I did still reflects the thing the code does. I am sure, though that further changes are certainly not worth efforts spent on it